src/modules/CreateChannel/components/InviteUsers/index.tsx

```diff
@@ -46,10 +47,11 @@ const InviteUsers: React.FC<InviteUsersProps> = ({ onCancel, userListQuery, show
     // @ts-ignore
     if (!applicationUserListQuery?.isLoading && !!applicationUserListQuery?.hasNext) {
       applicationUserListQuery.next().then((users_) => {
+        if ('filterFn' in applicationUserListQuery) users_ = users_.filter((user) => applicationUserListQuery.filterFn(user));
         setUsers(users_);
       });
     }
-  }, [userQueryCreator?.hasNext]);
+  }, [userQueryCreator]);
```

Comment:
Why did you remove .hasNext?

Reply:
Porque solo actualizaba cuando el valor de hasNext cambiaba y no cuando los parámetros de userQueryCreator cambiaban. Entonces cuando el nuevo campo del filtro de nickNameStartsWith cambiaba, el useEffect no volvía a ejecutarse porque hasNext como tal no había cambiado